tests/test_mlip_calculators.py

```diff
@@ -49,6 +49,10 @@
 
 NEQUIP_PATH = MODEL_PATH / "toluene.nequip.pth"
 
+ALPHANET_PATH = MODEL_PATH / "alphanet"
+ALPHANET_CKPT = ALPHANET_PATH / "test_model.ckpt"
```

Comment:
Is there a way to download this as part of the test (normally to the local cache)? That is what we do for quite a few of the models, and ideally there should be some sort of function built into AlphaNet to do this by the relevant label.

Reply:
in theory we can use the same approach from data-tutorials if there is no option on their side